Setup/CreatesWebpayOrdersTable.php

```diff
@@ -22,10 +22,10 @@ protected function createWebpayOrdersTable(SchemaSetupInterface $setup)
             ->addColumn('order_id', Table::TYPE_TEXT, 60, [
                 'nullable' => false,
             ], 'Order Id')
-            ->addColumn('buy_order', Table::TYPE_TEXT, 20, [
+            ->addColumn('buy_order', Table::TYPE_TEXT, 26, [
                 'nullable' => false,
             ], 'Buy order')
-            ->addColumn('child_buy_order', Table::TYPE_TEXT, 20, [
+            ->addColumn('child_buy_order', Table::TYPE_TEXT, 26, [
```

Comment:
These changes will not affect active installations. You need to use Patch Schema.
Check this: https://developer.adobe.com/commerce/php/development/components/declarative-schema/patches/

Reply:
it's here: https://github.com/TransbankDevelopers/transbank-plugin-magento2-webpay-rest/blob/feat/add-generate-buyorder/etc/db_schema.xml

Reply:
true